tests/domain/Shared/test_tax_id_number.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+
+from erica.domain.Shared.tax_id_number import TaxIdNumber
+
+
+class TestTaxIdNumberValidation:
+
+    def test_valid_id_nr_returns_no_validation_error(self):
+        tax_id_number = "04452397687"
+        TaxIdNumber.validate_tax_id_number(tax_id_number)
```

Comment:
Can you add a test for a real tax number?

Reply:
Yes, these were the tests from steuerlotse